lib/src/endpoints/me.dart

```diff
@@ -34,12 +34,19 @@ class Me extends EndpointPaging {
     return Player.fromJson(map);
   }
 
-  Future<Iterable<Track>> recentlyPlayed() async {
-    var jsonString = await _api._get('$_path/player/recently-played');
-    var map = json.decode(jsonString);
-
-    var items = map['items'] as Iterable<dynamic>;
-    return items.map((item) => Track.fromJson(item['track']));
+  Future<Iterable<PlayHistory>> recentlyPlayed(
```

Comment:
Since the endpoint does support it, wouldn't it make sense to use paging here? 🤔

Reply:
It's a bit more complex because the paging here uses dates instead of index offset, so @nrubin29 would have to override the default paging behavior.
But it should be done.

Reply:
We'll add correct paging later, let's pull it in for the _value_ ;)